htdocs/adherents/type.php

```diff
@@ -399,9 +399,8 @@
 	print dol_get_fiche_end();
 
 	print '<div class="center">';
-	print '<input type="submit" name="button" class="button" value="'.$langs->trans("Add").'">';
-	print '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;';
-	print '<input type="submit" name="cancel" class="button button-cancel" value="'.$langs->trans("Cancel").'" onclick="history.go(-1)" />';
+	print '<input type="submit" class="button button-save" name="save" value="'.$langs->trans('Add').'">';
```

Comment:
If my understanding is good, we can replace this portion of code with

print $form->buttonsSaveCancel();